### DISCUSSION TITLE
Vault switcher not switching vaults.

### DISCUSSION THREAD

**Discussion by Sinled** [chrisgrieser/shimmering-obsidian#150] [Q&A] [answered]
Hi, am i missing something or "vault switcher" not switching vaults? 
At the moment it only opens vault in obsidian but search continues to work only in vault specified in workflow config.

**Comment by chrisgrieser** [comment, marked as answer]
Hi, in previous versions (before 3.13), there was such a feature to change the vault you control/search via the vault switcher. However, for compatibility reasons with Alfred 5's new workflow configuration, I had to remove that. Instead, you have to select a different vault in the Alfred workflow configuration.

**Reply by Sinled** [reply]
Thanks for reply, any chances that this feature will return in some form other than configuration? Switching via configuration is not very convenient when you have 2+ equally used vaults. Shouldn't it be possible via alfred env variables?

**Reply by chrisgrieser** [reply]
yeah, it's fine if you occasionally switch the vault to search, but not if you wanna do it frequently, that's true.

Unfortunately, with Alfred 5's configuration, this is not as easy to implement (if at all). If you can open up a feature request and there is general interest in that, I can try to look into it
